frontend/src/components/RecentOperations/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useCallback, useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { Table } from "@stellar/design-system";
+
+import { useRedux } from "hooks/useRedux";
+import { networkConfig } from "constants/settings";
+import { SectionCard } from "components/SectionCard";
+import { AccountBadge } from "components/AccountBadge";
+import { OperationTypeColumn } from "./OperationTypeColumn";
+import { fetchLastOperationsAction } from "ducks/operations";
+import { FetchLastOperationsActionResponse, Network } from "types";
+
+import "./styles.scss";
+
+export const RecentOperations = () => {
+  const { operations } = useRedux("operations");
+  const dispatch = useDispatch();
+  const horizonURL = "https://horizon.stellar.org";
+
+  const labels = [
+    {
+      id: "source",
+      label: "Source",
+    },
+    {
+      id: "operations",
+      label: "Operations",
+    },
+    {
+      id: "details",
+      label: "Details",
+    },
+    {
+      id: "timeAgo",
+      label: "Time ago",
+    },
+  ];
+
+  const getLastOperations = useCallback(() => {
+    dispatch(fetchLastOperationsAction());
+  }, [dispatch]);
+
+  useEffect(() => {
+    getLastOperations();
+    setInterval(getLastOperations, 10 * 1000);
+  }, [dispatch, getLastOperations]);
+
+  const renderOperations = (operation: FetchLastOperationsActionResponse) => {
+    const operationLink = `${horizonURL}/operations/${operation.id}`;
+
+    return (
+      <React.Fragment key={Math.random()}>
```

Comment:
Let's not use `Math.random()` for the key here for better performance. [React docs](https://reactjs.org/docs/reconciliation.html) say this:

> Keys should be stable, predictable, and unique. Unstable keys (like those produced by Math.random()) will cause many component instances and DOM nodes to be unnecessarily recreated, which can cause performance degradation and lost state in child components.